### PR TITLE
Add a note about installting kubectl.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #308, #279

### CODE REVIEW

**Comment by jimmidyson** [comment]
Reckon we should offer to download kubectl on start if it's not available in PATH to directory of user's choosing.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a
 * Linux
     * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [KVM](http://www.linux-kvm.org/) installation, 
 * VT-x/AMD-v virtualization must be enabled in BIOS
+* `kubectl` must be on your path. Minikube currently supports any version of `kubectl` greater than 1.0, but we recommend using the most recent version.
+  You can install kubectl with [these steps](https://coreos.com/kubernetes/docs/latest/configure-kubectl.html).
```
Comment (jimmidyson) [inline README.md]: Should we point to kubernetes docs at http://kubernetes.io/docs/getting-started-guides/docker/#download-kubectl instead?

**Comment by dlorenc** [comment]
Yeah, we should just install it automatically. We have #83 tracking that.

**Comment by dlorenc** [comment]
> Should we point to kubernetes docs at http://kubernetes.io/docs/getting-started-guides/docker/#download-kubectl instead?
>  Add a line note

Yeah switched it over to the minikube ones.

**Comment by jimmidyson** [comment]
LGTM

**Comment by luxas** [comment]
LGTM

**Pull request merged**